test/spectral/tests/spectral-config.test.js

```diff
@@ -1,37 +1,23 @@
-const path = require('path');
```

Comment:
It looks like you snuck some unrelated changes into this PR.  I don't have a problem with that but I think you should alert reviewers to this by either mentioning it in the PR description or simply commenting on your own PR to call this out.

Reply:
Yes, I will do a better job of pointing out and explaining these other changes. These tests were timing out unpredictably (maybe 1/10 or 1/20 times). The mocking seemed to cause the problems, so I removed the mocking to simplify the tests and fix the timeouts.